bootstrap/pkg/kfapp/coordinator/coordinator.go

```diff
@@ -618,10 +618,16 @@ func (kfapp *coordinator) Show(resources kftypes.ResourceEnum, options map[strin
 			} else {
 				return &kfapis.KfError{
 					Code: int(kfapis.INTERNAL_ERROR),
-					Message: fmt.Sprintf("%v not in Platforms",
+					Message: fmt.Sprintf("coordinator Show failed for %v: Not support 'Show'",
```

Comment:
s/support/supported

Reply:
I think `Not support 'Show'` or `'Show' not supported`